docs/source/quick_start/examples/BSTLD.rst

```diff
@@ -1,18 +1,22 @@
-########
+
```

Comment:
these rst files should be removed.

Reply:
Ok, there's actually one problem, these files will be generated automatically when we use `make html`, but there is no easy way to remove them like  `make clean`.

Reply:
We need another PR for `make clean`.

What I mean is these files should be removed from git.
Generated files should not be managed by git.